app/models/manageiq/providers/hawkular/middleware_manager/event_catcher/runner.rb

```diff
@@ -1,20 +1,21 @@
 class ManageIQ::Providers::Hawkular::MiddlewareManager::EventCatcher::Runner <
   ManageIQ::Providers::BaseManager::EventCatcher::Runner
 
+  TAG_EVENT_TYPE    = "miq.event_type".freeze # required by fetch
+  TAG_RESOURCE_TYPE = "miq.resource_type".freeze # optionally provided when linking to a resource
+
   def initialize(cfg = {})
     super
 
-    # Accept events with an event category in the keyset and whose event message matches the (optional) pattern value
-    # - key = event category
-    # - value (optional) event message regex
-    @whitelist = {
-      'Hawkular Deployment' => nil
-    }.freeze
-
-    # Some ems events will link to objects in miq inventory. This maps the event category to the object type
-    @types = {
-      'Hawkular Deployment' => MiddlewareServer.name
-    }.freeze
+    # Supported event_types (see settings.yml)
+    @whitelist = [
+      # summary
+      'hawkular_deployment.error',
```

Comment:
I group the summary and detail event_types together, each of those sublists are alphabetical.